themes/ansible-community/templates/base_footer.tmpl

```diff
@@ -21,7 +21,7 @@
         <div class="footer-right">
           <span class="redhat">Sponsored by</span>
           <span class="redhat-logo">
-            <a href="https://www.redhat.com/" target="_blank">
+            <a href="{{ homepage.links.platform.home }}" target="_blank">
```

Comment:
Let me check if the supported by RH link should remain on redhat.com

Reply:
There's no hard and fast rule where the endorsement link has to point to, so let's keep it this way for now and modify if needed.